canhttp/Cargo.toml

```diff
@@ -24,10 +27,12 @@ thiserror = { workspace = true }
 
 [dev-dependencies]
 candid = { workspace = true }
+itertools = { workspace = true }
 maplit = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 
 [features]
 default = ["http"]
 http = ["dep:http", "dep:num-traits", "dep:tower-layer"]
-json = ["http", "dep:serde", "dep:serde_json"]
\ No newline at end of file
+json = ["http", "dep:serde", "dep:serde_json"]
+multi = ["dep:ciborium", "dep:sha2", "dep:futures-channel"]
```

Comment:
Understanding question: why is this not the default? Also, what does futures-channel achieve that previously cannot be achieved by using plain vectors?